terraform/modules/happy-stack-eks/variables.tf

```diff
@@ -77,8 +78,9 @@ variable "services" {
     synthetics : optional(bool, false),
     initial_delay_seconds : optional(number, 30),
     period_seconds : optional(number, 3),
-    platform_architecture : optional(string, "amd64"), // Supported values: amd64, arm64
-    bypasses : optional(map(object({                   // Only used for INTERNAL service_type
+    platform_architecture : optional(string, "amd64"),     // Supported values: amd64, arm64; GPU nodes are amd64 only.
+    additional_node_selectors : optional(map(string), {}), // For GPU use: { "nvidia.com/gpu.present" = "true" }
```

Comment:
I think having this generally. Maybe in addition, should we add the nvidia specific node selectors if gpu is not null so you don't have to know to add both? Maybe also override the platform_architecture if gpu is not null

Reply:
Done!